.github/workflows/build.yml

```
@@ -10,9 +10,10 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy: 
+    strategy:
+      fail-fast: false
       matrix:
-        dir: [ 'jdk', 'oscript' ]
+        script: [ 'build-base-k8s-jenkins-agent.sh', 'build-base-k8s-jenkins-agent.sh', 'build-edt-k8s-agent.sh', 'build-edt-swarm-agent.sh', 'build-oscript-k8s-agent.sh', 'build-oscript-swarm-agent.sh', 'build-server.sh' ]
```

actionlint

duplicate value "build-base-k8s-jenkins-agent.sh" is found in matrix "script". the same value is at line:16,col:19

(matrix)